include/zephyr/bluetooth/services/ans.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2025 Sean Kyer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+
+/**
+ * @brief Alert Notification Service (ANS)
+ * @defgroup bt_ans Alert Notification Service (ANS)
+ * @ingroup bluetooth
+ * @{
+ */
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief ANS max text string size
+ *
+ * This is the max string length to be saved in a New Alert. Text longer than the max is truncated.
+ */
+#define BT_ANS_MAX_TEXT_STR_SIZE 18
+
+/**
+ * @brief ANS Category ID Type
+ */
+typedef uint8_t bt_ans_cat_t;
+
+/**
+ * @brief ANS Category ID Enum
+ *
+ * Enumeration for whether the category is supported.
+ */
+enum {
+	bt_ans_cat_simple_alert,
+	bt_ans_cat_email,
+	bt_ans_cat_news,
+	bt_ans_cat_call,
+	bt_ans_cat_missed_call,
+	bt_ans_cat_sms_mms,
+	bt_ans_cat_voice_mail,
+	bt_ans_cat_schedule,
+	bt_ans_cat_high_pri_alert,
+	bt_ans_cat_instant_message,
+	/* Marker for number of categories */
+	bt_ans_cat_num,
+	/* 10–15 reserved */
```

Comment:
I don't know if we actually have any rules about this, but most (all?) enums in Zephyr are in caps, like `BT_ANS_CAT_SIMPLE_ALERT` instead of `bt_ans_cat_simple_alert`

Reply:
Yeah, shouldn't be too difficult to verify that values between `enum {` and `};` are caps

https://docs.zephyrproject.org/latest/contribute/style/code.html states that we follow the Linux style guide, and they do actually state (https://www.kernel.org/doc/html/v4.10/process/coding-style.html#macros-enums-and-rtl)

```
12) Macros, Enums and RTL

Names of macros defining constants and labels in enums are capitalized.
```

So we do have a rule for it, but seemingly it isn't checked.

Reply:
Just FYI, created https://github.com/zephyrproject-rtos/zephyr/issues/95679 to fix this :)